HW456/HW56/Accounts/IncreaseAccount.cs

```diff
@@ -0,0 +1,79 @@
+namespace HW456.HW5.Accounts;
+
+public class IncreaseAccount
+{
+    public string AccountNumber { get; set; }
+    public string OwnerName { get; set; }
+    public virtual decimal Balance { get; set; }
+    public bool IsActive { get; set; }
+
+    public virtual bool IncreaseBalance(decimal sum)
+    {
+        if (IsActive)
+        {
+            Balance += sum;
+            return true;
+        }
+        else
+        {
+            Console.WriteLine("Operation is not available");
+            return false;
+        }
+    }
+    public virtual void DecreaseBalance(decimal sum)
+    {
+        if (IsActive)
+        {
+            if (sum > Balance)
+            {
+                Console.WriteLine("The amount of decreasing is bigger then the balance");
+            }
+
+            else
+            {
+                Balance -= sum;
+            }
+
+        }
+        else
+        {
+            Console.WriteLine("Operation is not available");
+        }
+    }

```

Comment:
Здесь так же можно понизить вложенность:
```suggestion
    public virtual void DecreaseBalance(decimal sum)
    {
        if (!IsActive)
        {
            Console.WriteLine("Operation is not available");
            return;
        }

        if (sum > Balance)
        {
            Console.WriteLine("The amount of decreasing is bigger then the balance");
            return;
        }

        Balance -= sum;
    }
```

Reply:
Там и ниже есть места, где тоже можно так сделать

Reply:
попраивла